app/locales/it/overlap.json

```diff
@@ -0,0 +1,11 @@
+
+{
+  "overlap_title": "Controlla Sovrapposizioni",
+  "overlap_para_1": "Le aree verdi sono dati aggregati della tua posizione, clicca per ingrandire. Gli indicatori rossi sono le tue posizioni registrate. \n\nClicca il pulsante per scaricare informazioni pubbliche su casi confermati, indicati come aree viola.",
```

Comment:
What is the difference between GREEN and RED? IT is not clear in this text

Reply:
Not clear in the English text either, tbh...